src/Relude/Applicative.hs

```diff
@@ -30,6 +32,17 @@ import Control.Applicative (Alternative (..), Applicative (..), Const (..), ZipL
 pass :: Applicative f => f ()
 pass = pure ()
 
+{- | For chaining applicative operations in forward applications using '(&)'
+
+>>> Just (+ 1) & appliedTo (Just 2)
+Just 3
+>>> Nothing & appliedTo (Just 2)
+Nothing
+-}
```

Comment:
Let's add fixity to named function according to it's operator version
```suggestion
-}
infixl 4 `appliedTo`
```

Reply:
Actually I was thinking about this, and I'm not sure if we should. If we use (&) for using these functions the last argument becomes the first: 

Just (* 2) & appliedTo (Just 3)

While if we use it as an infix function it becomes the first: 

Just (* 2) `appliedTo` Just 3

I like the (&) approach because it helps a lot newcomers to see the flow of data. 😄

Reply:
@NickSeagull Okay, makes sense for now :+1: